### ISSUE TITLE
Scenario QTs not being started with correct time remaining

### ISSUE THREAD

**Issue by warrensearle** [jac-uk/apply#592]
There is an issue with the Timer or the calculations made when starting a QT.

A number of candidates in the Scenario QT on 4 Nov reported incorrect timer durations:
- One candidate should have had 60 minutes to complete the test, but had 28 seconds remaining as soon as they started the test. They did this twice (on the same device)
- Another candidate should have had 1 hour 15 minutes to complete the test, but only had 1 hour and 8 minutes
- Another candidate reported having 2 hours to complete the test when they should have had 60 minutes
- [ any more? ]

**Comment by lloback** [comment]
I will try to run a test on a different timezone, and / or different time on the computer. I will post the comments here.

**Comment by warrensearle** [comment]
We need to make sure all timestamps are set using the following:
```
firebase.firestore.FieldValue.serverTimestamp()
```

In a few places our code is using `firebase.firestore.Timestamp.fromDate(new Date())` which relies on local computer date

**Comment by warrensearle** [comment]
So we need to make sure the Countdown component being used is not effected by local computer time settings.

**Comment by mbrookeswebdev** [comment]
I've experimented quite a bit with changing time on my laptop backwards and forwards and thought I'd share my observations.

If the time was adjusted backwards the candidate was kicked out of the test, but the timer was not working at all. This would've been noted by the candidate and logs didn't show up either.

I tried twice with the time adjusted forwards, the first time the timer worked fine and the candidate was given an extra hour, however when the test was submitted it didn't show that the candidate worked an hour more than they should've done. I tried again and the timer was fine (I think maybe something to do with caching). On submission there was no issues other than start/end dates on admin side were showing correct times and the Questions log part said the candidate started an hour earlier.
